db/migrate/20200707182629_create_reviews.rb

```diff
@@ -0,0 +1,12 @@
+class CreateReviews < ActiveRecord::Migration[5.1]
+  def change
+    create_table :reviews do |t|
+      t.string :title
+      t.integer :rating
+      t.text :content
+      t.string :image
```

Comment:
weird to think of an image as a string, but the URL is a string which is where the data type comes from.